### ISSUE TITLE
Offset and limit combined with aggregation (group by time) gives incorrect results

### ISSUE THREAD

**Issue by pjmyburg** [influxdata/influxdb#6661]
### Bug report

__System info:__ InfluxDB 0.13.0, Amazon Linux AMI on EC2

__Steps to reproduce:__

1. Insert many data points with timestamps over multiple days
2. Query count of data points and group by time(1d)
3. Query count of data points and group by time(1d) and offset 1
4. Query count of data points and group by time(1d) and limit 1

__Expected behavior:__ 

- I expect the first entry in the result of step 3 to match the second entry in the result of step 2.
- I expect the result of step 4 to match the first entry in the result of step 2.

__Actual behavior:__ 

_Step 2:_
```
select count(field) from measure where userId = 'abc' and time >= '2016-05-01' group by time(1d)
```
time		|	count
------------- | -------------
1462060800000000000	|45884
1462147200000000000	|83958
1462233600000000000	|79540
... |...

_Step 3:_
```
select count(field) from measure where userId = 'abc' and time >= '2016-05-01' group by time(1d) offset 1
```
time		|	count
------------- | -------------
1462147200000000000	|2
1462233600000000000	|0
1462320000000000000	|0
... |...

_Step 4:_
```
select count(field) from measure where userId = 'abc' and time >= '2016-05-01' group by time(1d) limit 1
```
time		|	count
------------- | -------------
1462060800000000000	|2

As you can see, the timestamps are correctly returned according to the limit and offset, but for some reason the count is also affected.

__Additional info:__ 

We just migrated from InfluxDB 0.9.5 to 0.13.0.  All steps were followed and the migration was a success.  The queries were working on 0.9.5 this morning, but not on 0.13.0.  We don't know which version in between resulted in the changed behaviour.
Offset and limit gives the expected behaviour when querying the raw data without aggregation, but gives incorrect results when any aggregation function is applied.

**Comment by jsternberg** [comment]
I think this is caused by #6391.

**Comment by pjmyburg** [comment]
Thanks guys!  Any realistic chance of a 0.13.1 with the fix soon or should we roll back to 0.12?

**Comment by jsternberg** [comment]
I'm going to try and put together the bugfixes that will go into 0.13.1 soon, but if you need this urgently, I personally would downgrade to 0.12 temporarily until we get 0.13.1 out.
